testplans/lotus-soup/go.mod

```diff
@@ -8,19 +8,20 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/drand/drand v1.2.1
 	github.com/filecoin-project/go-address v0.0.5
-	github.com/filecoin-project/go-data-transfer v1.1.1-0.20210428151930-29bfef7e037e
+	github.com/filecoin-project/go-data-transfer v1.4.3
```

Comment:
@dirkmc Shouldn't this be v1.5.0 ?

Reply:
This is the go.mod for lotus-soup so the deps just need to match the lotus dep.
The lotus dep is v1.9.0, which depends on go-data-transfer v1.4.3:
https://github.com/filecoin-project/lotus/blob/v1.9.0/go.mod#L35